docs/integrations/pytorch_lightning.rst

```diff
@@ -53,7 +53,7 @@ You have ``Python 3.x`` and following libraries installed:
 
 * ``neptune-client==0.4.123`` or newer: See :ref:`neptune-client installation guide <installation-neptune-client>`.
```

Comment:
I think (if the examples didn't fail) it works with neptune-client==0.4.126

Reply:
fixed